src/snyk/common/languageServer/languageServer.ts

```diff
@@ -82,7 +82,7 @@ export class LanguageServer implements ILanguageServer {
         configurationSection: CONFIGURATION_IDENTIFIER,
       },
       middleware: new LanguageClientMiddleware(this.configuration),
-      outputChannel: this.window.createOutputChannel(SNYK_LANGUAGE_SERVER_NAME),
+      outputChannel: this.client?.outputChannel ?? this.window.createOutputChannel(SNYK_LANGUAGE_SERVER_NAME),
```

Comment:
Under which condition the output channel can still be present there?

When we call `stop(), this will also cleanup the output channel per https://github.com/microsoft/vscode-languageserver-node/blob/cdf4d6fdaefe329ce417621cf0f8b14e0b9bb39d/client/src/common/client.ts#L3370. I don't think reusing the output channel that is meant to be disposed is a good idea.

Reply:
Nice! Didn't dig that far 👍 Can you please document this in a comment? Also make sure to specify the language-client version it's relevant to.